README.md

```diff
@@ -10,11 +10,11 @@ This project is a [_fastlane_](https://github.com/fastlane/fastlane) plugin. To
 fastlane add_plugin dynatrace
 ```
 
-Make sure you have the latest version of the DSS client (bundled with the agent). You can download it here [Latest iOS Agent](https://downloads.dynatrace.com/clientservices/agent?version=latest&techtype=ios)
+Make sure you have the latest version of the DSS client, which is bundled with the agent. You can download it from here [Latest iOS Agent](https://downloads.dynatrace.com/clientservices/agent?version=latest&techtype=ios)
```

Comment:
for managed this can cause problems - we do not yet have a dedicated download link where you can pass teh cluster version to get the lates compatible DSS e.g. in case of managed

Reply:
What do you suggest then?

Reply:
Added notice